keps/sig-network/2079-network-policy-port-range/README.md

```diff
@@ -282,10 +302,10 @@ _For GA, this section is required: approvers should be able to confirm the
 previous answers based on experience in the field._
 
 * **Will enabling / using this feature result in any new API calls?**
-  TBD
+  No
 
 * **Will enabling / using this feature result in introducing new API types?**
-  No, unless the new `EndPort` is considered a new API type
+  No
```

Comment:
nit: L318 - it's actually more than two bytes (e.g. you need also field name/number in its serialized format). [note that it matters much - just nitpicking]